contrib/scripts/go-mod-tidy-all.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+for modfile in $(find . -name go.mod); do
+ echo "Updating $modfile"
+ DIR=$(dirname $modfile)
+ (cd $DIR; go mod tidy)
```

Comment:
```suggestion
 (cd $DIR; go mod tidy --compat=1.17)
```

Reply:
Although this looks like it won't be needed once go version in our go.mod is upgraded to 1.18